docs/sentiment-analysis.md

```diff
@@ -0,0 +1,8 @@
+Sentiment analysis is the process of using text analysis to
+identify and study states of emotion in regards to the input, which is subjective information.
+
+Sentiment analysis is able to identify the amount of sentiment each user has
+for what it is processing. It will give numbers based on each users sentiment.
+
+To each user, outputs of higher numbers mean a higher sentiment and lower numbers are given
+for lower sentiment. Then the numbers are graphed according to each user.
```

Comment:
what is `higher sentiment` and `lower sentiment` do they translate to positive and negative wording/phrases. If so, maybe that should be included to increase clarity. Also, it might be helpful to mention the range of values that sentiment is scored on, is it -1: negative, 0: neutral, 1: positive, or does it just range from 0-1. For a user that has no experience with sentiment analysis, this can be useful to know.